components/CheckDocumentation/vacuum/MxidWraparound.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const MxidWraparoundTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const warning = config.settings["warning_threshold_pct"] as number;
+  const critical = config.settings["critical_threshold_pct"] as number;
+  return (
+    <p>
+      Detects when the multixact ID utilization on the server is more than{" "}
```

Comment:
```suggestion
      Detects when multixact ID utilization on the server is more than{" "}
```

Reply:
per sync, I'll use "multixact ID space utilization" here too, I think it's better to use the same terminology here